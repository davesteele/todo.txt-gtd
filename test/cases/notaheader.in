# Aproj
#
# notaheader
#
#

task @computer +Aproj
