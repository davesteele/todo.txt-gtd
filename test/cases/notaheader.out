# Aproj
#
# notaheader
#
#

task @computer +Aproj

# _None
#

